nautobot/extras/jobs.py

```diff
@@ -1153,9 +1154,11 @@ def run_job(self, job_class_path, *args, **kwargs):
     job.request = self.request
     try:
         job.before_start(self.request.id, args, kwargs)
+        publish_event(topic="nautobot.events.jobs.started", payload=kwargs)
         result = job(*args, **kwargs)
         job.on_success(result, self.request.id, args, kwargs)
         job.after_return(JobResultStatusChoices.STATUS_SUCCESS, result, self.request.id, args, kwargs, None)
+        publish_event(topic="nautobot.events.jobs.finished", payload=kwargs)
```

Comment:
Just from a self-consistency standpoint it might make sense for `started` event to come before `before_start()` since `finished` comes after `after_return()`?